provision/infinispan/create_ispn_clusters.sh

```diff
@@ -214,6 +214,38 @@ spec:
 EOF
 }
 
+function deploy_replicated_cache_cr() {
```

Comment:
Isn't this effectively the same as `deploy_distributed_cache_cr` except that we specify `replicatedCache`? I think it would be simpler to just make the cache type a parameter of the original `deploy_cache_cr` method so that we  don't have to maintain duplicated code.

Reply:
@ryanemerson there is also the number of owners. I would say we can leave this as is now, as we plan to migrate to Helm anyway. WDYT?

Reply:
Yeah that's fine, it was just a nitpick :+1: